src/api/DummyApiResponder.php

```diff
@@ -1085,6 +1085,7 @@ protected function get_interface_response_loadPlayerInfo() {
                                 'gender' => '',
                                 'image_size' => NULL,
                                 'autopass' => TRUE,
+                                'fire_overshooting' => TRUE,
```

Comment:
Regarding your comments in #1704, do you want this to be set to FALSE instead? It doesn't actually change any of the current unit tests, but it would if we were to add a fire overshooting test.